Pod/Classes/TUSResumableUpload.m

```diff
@@ -71,10 +77,26 @@ - (void) start
 - (void) createFile
 {
     [self setState:CreatingFile];
-
-    NSUInteger size = [[self data] length];
-    NSDictionary *headers = @{ HTTP_FINAL_LENGTH: [NSString stringWithFormat:@"%u", size] } ;
-    NSMutableURLRequest *request = [[NSMutableURLRequest alloc] initWithURL:[self endpoint] cachePolicy:NSURLRequestReloadIgnoringLocalCacheData timeoutInterval:REQUEST_TIMEOUT];
+    
+    NSUInteger size = (NSUInteger)[[self data] length];
+    
+    NSMutableDictionary *mutableHeader = [NSMutableDictionary dictionary];
+    [mutableHeader addEntriesFromDictionary:[self uploadHeaders]];
+    [mutableHeader setObject:[NSString stringWithFormat:@"%lu", (unsigned long)size] forKey:HTTP_FINAL_LENGTH];
+    
+    [mutableHeader setObject:HTTP_TUS_VERSION forKey:HTTP_TUS];
+    NSString *plainString = @"LongLiveQuontr.mp4";
```

Comment:
Could you please either remove this hard-coded filename or make it configurable?